WcaOnRails/config/locales/en.yml

```diff
@@ -1459,6 +1459,12 @@ en:
         search: "Name or city"
         #context: the "recent" button
         recent: "Last %{count} days"
+        #context: the registration status icon
+        registration:
+          opens_in: "Registration will open in %{duration}"
+          closed: "Registration has closed"
+          open: "Registration is open!"
```

Comment:
What about `Registration is open until %{date}`? I'm debating whether this could be too "risky" in terms of users thinking they can wait a few more days while the registration is _technically_ still open, only to find out that the competitor limit has been reached.

Reply:
I would tend to agree with not having the days written in the tooltip. The large variability in how fast competitions fill up  make the effective closing date not just the registration closing time

Reply:
Good point. We can come back to this discussion later once the current version has been deployed and we have some feedback from the community.